manifests/service.pp

```diff
@@ -5,23 +5,24 @@
   $manage_oddjobd = $sssd::manage_oddjobd,
   $service_ensure = $sssd::service_ensure,
 ) {
-  service { $sssd_service:
-    ensure     => $service_ensure,
-    enable     => true,
-    hasstatus  => true,
-    hasrestart => true,
-  }
```

Comment:
Status/restart are usually defined by the provider. Having them here will cause conflicts in the general case since other modules don't generally explicitly define them.